st2mistral/functions/stackstorm.py

```diff
@@ -18,16 +18,104 @@
 from oslo_config import cfg
 from oslo_log import log as logging
 
-from st2mistral import config
-config.register_opts()
-
 from mistral import exceptions as exc
+from st2common.jinja.filters import crypto
```

Comment:
Sorry, I must have missed that one, or assumed that "duplicate" meant implementing these redirections. I am okay with re-implementing them fully here, it just adds work to anyone wanting to create a new custom filter.

Funny enough, I didn't catch the deployment complications you allude to because of the fact I am using one virtualenv for both st2 and mistral in https://github.com/StackStorm/st2/pull/3557 (which you pointed out should be done separately). I assume this is what you're talking about, since in any real deployment, mistral would not have access to `st2common`.

I'll adjust this PR's description and change the approach. Thanks for the catch!